### DISCUSSION TITLE
row number

### DISCUSSION THREAD

**Discussion by Mambi** [awcodes/filament-table-repeater#34] [General]
Hi,
How get the row number, please?

**Comment by awcodes** [comment]
There is no row numbers by default.

You could probably use a Placeholder and generate the row numbers yourself. If that's what you're asking.

**Comment by kylearch** [comment]
I know this is old but I was looking for an answer to this and came up with a workable, though maybe not the best, solution:

```
// There might be a different or better way to get the uuid of the repeater row than this, but this worked from a FormComponent inside the repeater schema.
$id = str($component->getContainer()->getStatePath())->explode('.')->last(); 
$row = array_search($id, array_keys($component->getParentRepeater()->getState()));
```
